rest_framework_latex/templatetags/rest_framework_latex.py

```diff
@@ -0,0 +1,35 @@
+from django import template
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.safestring import mark_safe
+
+register = template.Library()
+
+
+@register.filter(name='latex_safe', is_safe=True)
+def latex_safe(value):
+    # http://www.cespedes.org/blog/85/how-to-escape-latex-special-characters
+    value = value.replace('\\', '\\textbackslash{}')
+    value = value.replace('{', '\\{')
+    value = value.replace('}', '\\}')
+    value = value.replace('#', '\\#')
+    value = value.replace('$', '\\$')
+    value = value.replace('%', '\\%')
+    value = value.replace('^', '\\^')
+    value = value.replace('_', '\\_')
+    value = value.replace('~', '\\textasciitidle{}')
+    value = value.replace('\\textbackslash\\{\\}', '\\textbackslash{}')
```

Comment:
When `\\` is changed to `\\textbackslash{}`, the next two rules change it to `\\textbackslash\\{\\}` - the bottom rule fixes that.

It's less efficient but it gives the correct result.